ChaosMod/Util/Peds.h

```diff
@@ -55,3 +55,24 @@ inline Ped CreateHostilePed(Hash modelHash, Hash weaponHash, Vector3 *location =
 	SET_PED_FIRING_PATTERN(ped, 0xC6EE6B4C);
 	return ped;
 }
+
+inline void SetCompanionRelationship(Ped ped, const char* name)
```

Comment:
Considering we expose it to the Lua runtime we should accept a `const std::string&` instead

Reply:
Noted, didn't know that